test/Spec/Integration.hs

```diff
@@ -237,27 +244,31 @@ testValueAssertionsOrderCorrectness =
         wallet1 = 200_000_000
         wallet2 = 300_000_000
 
-        payFee = 146200 -- taken from trace
+        payFee = 146200
         payTo0Amt = 11_000_000
         payTo1Amt = 22_000_000
         payTo2Amt = 33_000_000
 
         wallet0After = wallet0 + payTo0Amt
         wallet2After =
-          wallet2 + payTo2Amt
+          wallet2
+            + payTo2Amt
             - payTo1Amt
             - payFee
+
         wallet1After =
-          wallet1 + payTo1Amt
+          wallet1
+            + payTo1Amt
             - payTo0Amt
             - payFee
             - payTo2Amt
             - payFee
      in assertExecution
           "Values asserted in correct order with withContractAs"
-          ( initAndAssertLovelace [wallet0] wallet0After
-              <> initAndAssertLovelace [wallet1] wallet1After
-              <> initAndAssertLovelace [wallet2] wallet2After
+          ( initCollateral $ -- Initialize all the wallets with the collateral utxo.
```

Comment:
Hmm. If it meant to be used this way, maybe name it `withCollateral`?

Reply:
I renamed it to `withCollateral`, actually every function in the `Init.hs` starts with `initSomething`, therefore I decided to name it that way.